Cargo.toml

```diff
@@ -18,15 +18,17 @@ tokio = { version = "1.40.0", features = ["full"] }
 toml = "0.8.19"
 toml_edit = "0.22.21"
 url = "2.5.2"
-wasmparser = { git = "https://github.com/bytecodealliance/wasm-tools" }
-wit-component = { git = "https://github.com/bytecodealliance/wasm-tools" }
-wit-parser = { git = "https://github.com/bytecodealliance/wasm-tools" }
+wasmparser = "0.219.1"
+wit-component = "0.219.1"
+wit-parser = "0.219.1"
 futures = "0.3.30"
 semver = "1.0.23"
-wit-bindgen-rust = { git = "https://github.com/fibonacci1729/wit-bindgen", branch = "deps" }
-wit-bindgen-core = { git = "https://github.com/fibonacci1729/wit-bindgen", branch = "deps" }
+wit-bindgen-rust = "0.34.0"
+wit-bindgen-core = "0.34.0"
 wasm-pkg-common = "0.5.1"
 wasm-pkg-client = "0.5.1"
+js-component-bindgen-component = { git = "https://github.com/bytecodealliance/jco" }
```

Comment:
Sanity check if this is still needed?  I can see where `-bindgen` is used but not `-bindgen-component`